SG Policy.md

```diff
@@ -40,13 +40,16 @@ G. As an exception, [Steering Group] decisions on confidential matters which hav
 
 A. The [Steering Group] may define policies and agreements that govern the operation of the WHATWG. Policies must not conflict with the [Steering Group Agreement]. If they do, the [Steering Group Agreement] takes precedence.
 
-B. Other than the [Steering Group Agreement] and the [WHATWG Principles], the [Steering Group] may adopt, amend, or revoke Policies or Agreements by [consensus]. The same applies to this Steering Group Policy.
+B. Other than the [Steering Group Agreement] and the [WHATWG Principles], the [Steering Group] may adopt, amend, or revoke policies or agreements by [consensus]. The same applies to this Steering Group Policy.
 
 C. The canonical copies of all WHATWG policies and agreements must be kept in the [Steering Group]'s repository. Policies may also be published elsewhere, such as on <https://whatwg.org/>.
 
-D. Changes to policies and agreements should be made by pull request associated with an issue. Changes must be reviewed by a [Steering Group Representative] and may only be merged by [Consensus] or [unanimous consent] of the [Steering Group], as set forth in the [Steering Group Agreement].
+D. Changes to policies and agreements should be made by pull request associated with an issue that has a title prefix of "Policy Change: ". Changes must be reviewed by a [Steering Group Representative] and may only be merged when:
 
-E. Anyone may propose policy changes via issue and pull request. The [Steering Group] is not obliged to respond to such proposals.
+* There is [consensus] or [unanimous consent] of the [Steering Group], as set forth in the [Steering Group Agreement].
+* For substantive non-urgent changes: 45 [days] have passed since review.
```

Comment:
Since we don't spell out what substantive or urgent means, should we instead carve out an exception to the rule with a few more words, saying that the rule is the normal case, but for trivial changes and urgent issues, the SG may make the changes on shorter notice?

Can we foresee an urgent but non-trivial change? If we are able to make such changes fast, would that not make the "Policy Change: " system less confidence-inspiring to participants? Should we promise to flag such issues in a special way so that people can react sooner to them?

Alternatively, we don't carve out an exception for urgency, and deal with that if it really comes up. It's hard (for me) to see a case where updating https://whatwg.org/policies can by itself fix an urgent problem.

Reply:
I wrote it that way because @othermaciej noted that concern in #116. I would be okay with changing it.

Reply:
@othermaciej do you have an example of what kind of change would be urgent, and how we should phrase the carve-out? The more narrow the exception is, the less concern anyone who really monitors for policy changes would have, I think.

Reply:
I'm going to go ahead and suggest dropping the exception here:

```suggestion
* 45 [days] have passed since review.
```

Reply:
The one thing I can think of that might be urgent is if we are trying to follow a policy process for the very first time and we notice it's broken. In practice though, most policy changes we've done have taken more than 45 days when timer deliberation is considered. I think an urgency exception would be useful just in case of emergency, and perhaps we could leave it to the SG's judgment which policy changes are urgent by explicitly saying so. I'd also tentatively be ok with no exception if that's simpler to draft.